### DISCUSSION TITLE
Lastpass as identity provider

### DISCUSSION THREAD

**Discussion by hallvoren** [smallstep/certificates#447] [Q&A] [answered]
Hi. Does anyone have any experience using lastpass as identity provider for step ca?

**Comment by mmalone** [comment, marked as answer]
`step-ca` uses a protocol called OAuth OIDC (OpenID Connect) for single sign-on. Lastpass seems to have a fairly extensive product offering these days. If they support that protocol then we should be able to integrate. I see some reference to single sign-on support on the lastpass website, but I can't find any technical documentation on what protocol(s) they support (the other common SSO protocol is SAML, which `step-ca` does not support). If you can find technical docs on the lastpass SSO offering I'd be happy to take a look and help as best I can!

**Reply by hallvoren** [reply]
Thanks, good to know. I also struggled to find much documentation. It is weekend here, but I will talk to some colleagues next week and see what I can find out.
